apps/cloudappclient/app.js

```diff
@@ -60,6 +61,11 @@ module.exports = activity => {
     }
   })
 
+  // for debug and test: save playerId map to property
+  pm.on('update', (handle) => {
+    property.set('app.cloudappclient.player', JSON.stringify(handle))
```

Comment:
AFAIK, there is a flora service registered in cloud app client. Maybe we could utilize it to declare a flora method to expose these debug informations.

Reply:
This is for automated test. Automated testing tools provide a convenient tool for reading data from getprop

Reply:
We do also provide a tool for invoking flora methods.

Reply:
already switch to Flora.